src/states/abstractmps.jl

```diff
@@ -80,4 +80,6 @@ TensorKit.spacetype(Ψtype::Type{<:AbstractMPS}) = spacetype(site_type(Ψtype))
 TensorKit.sectortype(Ψ::AbstractMPS) = sectortype(typeof(Ψ))
 TensorKit.sectortype(Ψtype::Type{<:AbstractMPS}) = sectortype(site_type(Ψtype))
 
+Base.isapprox(Ψ1::AbstractMPS, Ψ2::AbstractMPS; atol = 1e-12) = abs(dot(Ψ1,Ψ2)-1) < atol
```

Comment:
This might fail for states that are not normalized, or when there is an additional phase? Do we want psi to be approximately equal to psi *im?

Reply:
you're right. I think `abs(abs(dot(Ψ1,Ψ2)/(norm(Ψ1)*norm(Ψ2)-1)) < atol` solves the problem.

Reply:
resolved with af24f84ef85af2e28e271530f21f9a151091fcb3 ?